src/metabase/lib/metadata/result_metadata.cljc

```diff
@@ -229,14 +229,23 @@
           (assoc col :source (legacy-source col)))
         cols))
 
+(defn- remove-namespaced-options
```

Comment:
This was the only namespace using this function so I moved it here... I am working towards removing `metabase.legacy-mbql.util` entirely (all MBQL manipulation should be done on MBQL 5 with Lib)